anaconda_anon_usage/tokens.py

```diff
@@ -24,6 +27,41 @@ def version_token():
     return __version__
 
 
+@cached
+def system_token():
+    """
+    Returns the system/organization token. Unlike the other
+    tokens, it is desirable for this token to be stored in
+    a read-only/system location, presumably installed
+    The system/organization token can be stored anywhere
+    in the standard conda search path. Ideally, an MDM system
+    would place it in a read-only system location.
+    """
+    # Do not import SEARCH_PATH directly since we need to
+    # temporarily patch it for testing
+    for path in c_constants.SEARCH_PATH:
+        # Only consider directories where
+        # .condarc could also be found
+        if not path.endswith("/.condarc"):
+            continue
```

Comment:
To clarify this will ignore paths that end in `condarc.d`? https://github.com/conda/conda/commit/1b5652665e4af5ccbca0ef6bbbd2b356063cffe6 added this path originally and it's still supported, and I don't really see us moving away from it any time soon given that we likely have (but can't prove it yet) users of it.

Reply:
Yes, because those are always subdirectories of a conda-controlled parent. So for example, `/etc/conda/.condarc` and `/etc/conda/condarc.d` are both contained in `/etc/conda`. So there's no reason for `org_token` to go deeper in the directory tree than necessary.